.mailmap

```diff
@@ -54,13 +55,16 @@ Maxime Dénès <mail@maximedenes.fr>                 Maxime Dénès <maxime.dene
 Maxime Dénès <mail@maximedenes.fr>                 maximedenes <maximedenes@users.noreply.github.com>
 Stéphane Desarzens <8899730+Columbus240@users.noreply.github.com>     Columbus240 <8899730+Columbus240@users.noreply.github.com>
 Olivier Desmettre <desmettr@gforge>                desmettr <desmettr@85f007b7-540e-0410-9357-904b9bb8a0f7>
+Christian Doczkal <christian.doczkal@inria.fr>     Christian Doczkal <christian.doczkal@ens-lyon.fr>
 Damien Doligez <doligez@gforge>                    doligez <doligez@85f007b7-540e-0410-9357-904b9bb8a0f7>
 İsmail Dönmez <ismail-s@users.noreply.github.com>  Ismail <ismail-s@users.noreply.github.com>
 formalize.eth <formalize@protonmail.com>           ilya <ilya@localhost.localdomain>
 Andres Erbsen <andreser@mit.edu>                   Andres Erbsen <andres@kevix.co>
 Jim Fehrle <jim.fehrle@gmail.com>                  Jim <jfehrle@sbcglobal.net>
 Jim Fehrle <jim.fehrle@gmail.com>                  Jim Fehrle <jim.fehrle@gmail.com>
 Jim Fehrle <jim.fehrle@gmail.com>                  jfehrle <jfehrle@users.noreply.github.com>
+Jim Fehrle <jim.fehrle@gmail.com>                  Jim Fehrle <jfehrle@sbcglobal.net>
+Jim Fehrle <jim.fehrle@gmail.com>                  Jim Fehrle <jim.fehrle@gmail.com>
```

Comment:
What is happening here? Isn't it the identity? And wasn't it already there above?

Reply:
Hm, yeah, not sure.  I've removed the duplicate